lnplus/docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: $APP_LNPLUS_IP
+      APP_PORT: $APP_LNPLUS_PORT
+  web:
+    image: lightningnetworkplus/lnpclient:v0.1.10
+    restart: on-failure
+    stop_grace_period: 1m
+    volumes:
+      - ${APP_LIGHTNING_NODE_DATA_DIR}:/lnd:ro
+    environment:
+      RAILS_ENVIRONMENT: $RAILS_ENVIRONMENT
```

Comment:
This should read `$APP_LNPLUS_RAILS_ENVIRONMENT` to avoid risk of reading a wrong environment variable.

Reply:
OK. Updated. Thank you!